build.sbt

```diff
@@ -1,39 +1,26 @@
 import play.sbt.PlayImport.PlayKeys
 import play.sbt.routes.RoutesKeys.routesImport
 import sbt.Keys._
-import sbt.Tests.{Group, SubProcess}
 import sbt._
-import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, scalaSettings}
+import uk.gov.hmrc.DefaultBuildSettings
+import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, scalaSettings, oneForkedJvmPerTest}
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
 import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
 
 val appName = "tax-history"
 
-lazy val scoverageSettings = {
-  import scoverage.ScoverageKeys
-  Seq(
-    // Semicolon-separated list of regexs matching classes to exclude
-    ScoverageKeys.coverageExcludedPackages := "<empty>;Reverse.*;.*AuthService.*;modgiels/.data/..*;" +
-      "uk.gov.hmrc.taxhistory.auditable;uk.gov.hmrc.taxhistory.metrics;view.*;controllers.auth.*;filters.*;forms.*;config.*;" +
-      ".*BuildInfo.*;prod.Routes;app.Routes;testOnlyDoNotUseInAppConf.Routes;controllers.ExampleController;controllers.testonly.TestOnlyController",
-    ScoverageKeys.coverageMinimumStmtTotal := 90.00,
-    ScoverageKeys.coverageFailOnMinimum := true,
-    ScoverageKeys.coverageHighlighting := true,
-    Test / parallelExecution := false
-  )
-}
-
 lazy val microservice =
   Project(appName, file("."))
     .enablePlugins(SbtAutoBuildPlugin, play.sbt.PlayScala, SbtDistributablesPlugin)
```

Comment:
I've left in "Test / parallelExecution := false", not sure if that's pulled in from somewhere.

Reply:
It is pulled in from [here](https://github.com/hmrc/sbt-settings/blob/main/src/main/scala/uk/gov/hmrc/DefaultBuildSettings.scala#L87) which you get by enabling SbtAutoBuildPlugin. You can test it out by running the command `reload` when it has been removed to capture the changes on the sbt shell and then running `Test / parallelExecution`. As the the screenshot shows below:
<img width="359" alt="Screenshot 2023-07-12 at 16 55 50" src="https://github.com/hmrc/tax-history/assets/63714619/cbf31e17-6743-43fe-8b2b-48d0f0f0901d">

Reply:
In SbtAutoBuildPlugin in sbt-auto-build, it can be seen [here](https://github.com/hmrc/sbt-auto-build/blob/7ea42a0d49d22ad33864bba3b4f8e612c14a98ff/src/main/scala/uk/gov/hmrc/SbtAutoBuildPlugin.scala#L51-L53) where scalaSettings and defaultSettings are brought in and can be seen from the logged message when you run the tests which is why they can be removed.
<img width="496" alt="Screenshot 2023-07-12 at 17 05 10" src="https://github.com/hmrc/tax-history/assets/63714619/6aba1dce-7e2f-4abe-a83e-ced263c6881f">